uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-request-files.scss

```diff
@@ -0,0 +1,19 @@
+.chr_leave-request-files_list {
+  height: 150px;
+  overflow-y: auto;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.chr_leave-request-files_meta {
+  margin-bottom: 10px;
+}
+
+.chr_leave-request-files_upload {
+  border: dashed 1px darkgray;
```

Comment:
Well all these files are just moved from SSP, but I looked that changing it to `$gray-darker` is not making much difference in how it looks. So Done